src/components/FullArticle.jsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+import axios from "axios";
+
+const FullArticle = () => {
+  const [article, setArticle] = useState(null);
+  const { article_id } = useParams();
+  useEffect(() => {
+    const queryString = `https://nc-backend-ecsl.onrender.com/api/articles/${article_id}`;
+    axios
+      .get(queryString)
+      .then((response) => {
+        console.log(response.data.article);
+        return response.data.article;
+      })
+      .then((data) => {
+        setArticle(data);
+      });
+  }, [article_id]);
+
```

Comment:
This axios request could be extracted out into an api file where you can keep all your api requests 👀